src/ghazal_azadi_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,38 @@
+# First Part of the Question
+
+
+def letter_freq(txt):
+    txt = txt.lower()
+    letter_new = {}
+    i: letter
+    for i in txt:
+        letter_new[i] = txt.count(i)
+    return letter_new
+
+# Using Dictionary that holds unique "Key Values" pair.
+
+
+if __name__ == '__main__':
+    text = input('Please enter text to analyse: ')
+
+    frequencies = letter_freq(text)
+    for letter, count in frequencies.items():
+        print('{:3}{:10}'.format(letter, count))
+
+
+# Second Part of the Question
+def letter_freq(txt):
```

Comment:
You are overwriting an existing function here, which should always be avoided. This function is also the same as the previous with an extra `sorted` function call. You shouldn't create a same function just to add some feature.
Also,
> As an extra challenge, modify the code **printing the table** so that letters are printed in alphabetical order

`sorted` function should be in the main code, not here.
`for letter, count in sorted(frequencies.items()):`